main/lxc/lxc.initd

```diff
@@ -96,7 +97,7 @@ start() {
 		return
 	fi
 
-	rm -f /var/log/lxc/${CONTAINER}.log
+	rm -f "$logdir"/${CONTAINER}.log
```

Comment:
Why does it remove the previous log file anyway?

Reply:
if a container starts successfully nothing is logged